tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/ScaleDownRemoveSFTest.java

```diff
@@ -133,15 +115,8 @@ public void testScaleDownCheckSF() throws Exception {
       removeConsumer(0);
 
       //check
-      result = servers[1].queueQuery(sfQueueName);
-      AddressQueryResult result2 = servers[1].addressQuery(sfQueueName);
-      if ("true".equals(option)) {
-         assertFalse(result.isExists());
-         assertFalse(result2.isExists());
-      } else {
-         assertTrue(result.isExists());
-         assertTrue(result2.isExists());
-      }
+      Assert.assertFalse(servers[1].queueQuery(sfQueueName).isExists());
+      Assert.assertFalse(servers[1].addressQuery(sfQueueName).isExists());
 
```

Comment:
Perhaps here we can also test the sf been deleted on servers[2]:

      Assert.assertFalse(servers[2].queueQuery(SimpleString.toSimpleString(snfAddress)).isExists());
      Assert.assertFalse(servers[2].addressQuery(SimpleString.toSimpleString(snfAddress)).isExists());
(tried myself and it's passing)

Reply:
will do.. thanks!

Reply:
@gaohoward I don't understand... ScaleDownRemoveSFTEst only have 2 servers.. (0 and 1) and 1 is being removed, right?

Reply:
@gaohoward perhaps you could send a PR later because I can't find where to add the check.

I can only see 2 servers here, and one of them died, leaving us with just servers[1]. I guess I don't understand what you meant.